lite-uda-installer-archives-mapping.ttl

```diff
@@ -9461,8 +9461,9 @@ source:
     dcterms:identifier "sql_lt.taz" ;
```

Comment:
lines 31 and 41 also need to be updated

Reply:
So update then.

Reply:
I cannot reach them within this PR; they are further away than the few-lines margin GitHub allows from the lines that were touched by this PR.

I can make a second PR after this one is merged.

Or Hugh can update them within this PR, because he can reach the entirety of both files.